backend/tests/Feature/Models/EntryModelTest.php

```
@@ -0,0 +1,146 @@
+            $this->assertFalse($recentlyTrashed->is_deadline_approaching);
+
+            $model->deleted_at = now()->subDays($thresholdDays + 1);
+            $model->save();
+
+            $longTrashed = $model->fresh();
+            if ($longTrashed === null) {
+                $this->fail('Model was not found after refresh.');
+            }
+            $this->assertTrue($longTrashed->is_deadline_approaching);
+        }
+    }
```

PHPMD (2.15.0)

[error] The method test_is_deadline_approaching is not named in camelCase. (undefined)

(CamelCaseMethodName)